mountainduck/preferences.md

```diff
@@ -12,6 +12,8 @@ Preferences
 
 Save all mounted volumes when quitting to be restored while relaunching.
 
+If you enable *Save Workspace* and [*Enable Login Item*](#login-item) in the same named preferences tab and do not manually eject the volume prior to reboot it will reconnect after login.
```

Comment:
```suggestion
If you enable *Save Workspace* and [*Enable Login Item*](#login-item) within *Preferences → Login Item* and do not manually eject the volume prior to quitting Mountain Duck, it will reconnect after login.
```